### DISCUSSION TITLE
Can't start a VM on UTM v3.2.0

### DISCUSSION THREAD

**Discussion by 3DCoded** [utmapp/UTM#3965] [UTM Help (macOS)]
When I start a VM on UTM v3.2.0, it gives me an error dialog. When I use the workaround mentioned in #3941, it gives me the error `h: no such file or directory`

* UTM Version: 3.2.0
* OS Version: 12.3.1
* Intel or Apple Silicon? Apple Silicon M1

**Comment by osy** [comment]
Even though you did not provide debug.log as requested, I am going to guess from context clues that what you did was enter "highmem=on" into the additional args box rather than in the QEMU Machine Properties box.

**Comment by 3DCoded** [comment]
I put it into the QEMU Machine Properties box. It seems that it only happened with Linux VMs. It's working fine on a Windows VM. I'm testing a Linux VM now again.

**Comment by 3DCoded** [comment]
At first I put it into the additional args, then I removed it and put into the QEMU Machine Properties box. Maybe it didn't remove properly.
